packages/cipherstash-proxy/src/postgresql/frontend.rs

```diff
@@ -737,6 +737,16 @@ where
 
                 Ok(typed_statement)
             }
+            Err(EqlMapperError::InternalError(str)) => {
+                warn!(
```

Comment:
Might be able to drop this `warn` as I think the error should be logged as error higher in the stack?

Reply:
Leaving it here for now as Proxy currently won't report mapping errors unless `enable_mapping_errors` is set